dartagnan/src/main/java/com/dat3m/dartagnan/program/processing/compilation/VisitorBase.java

```diff
@@ -97,52 +98,52 @@ public List<Event> visitUnlock(Unlock e) {
 
     @Override
 	public List<Event> visitStoreExclusive(StoreExclusive e) {
-		throw new IllegalArgumentException("Compilation for " + e.getClass().getName() + " is not supported by " + getClass().getName());
+		throw new IllegalArgumentException("Compilation for " + e.getClass().getSimpleName() + " is not supported by " + getClass().getSimpleName());
 	};
 
 	@Override
 	public List<Event> visitRMWAbstract(RMWAbstract e) {
-		throw new IllegalArgumentException("Compilation for " + e.getClass().getName() + " is not supported by " + getClass().getName());
+		throw new IllegalArgumentException("Compilation for " + e.getClass().getSimpleName() + " is not supported by " + getClass().getSimpleName());
```

Comment:
Maybe it would be nicer to add a method `throwError(Event)` that generates the error message rather than copying it over and over again.